NuGet.pkg.props

```diff
@@ -0,0 +1,31 @@
+<Project>
+  <Import Project="SourceLink.props" />
+
+  <!-- Publishing: Package Properties -->
+  <PropertyGroup>
+    <Title>$(AssemblyName)</Title>
+    <Authors>Cory Bennett</Authors>
+    <Owners>Videre Project</Owners>
+    <Copyright>Copyright (c) $([System.DateTime]::UtcNow.ToString(yyyy)) $(Authors)</Copyright>
+    <PackageIcon>icon.png</PackageIcon>
+    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+  </PropertyGroup>
+  <ItemGroup>
+    <None Include="$(SolutionDir)\assets\icon.png" Pack="true" PackagePath="" />
+    <None Include="$(SolutionDir)\NOTICE"          Pack="true" PackagePath="" />
+  </ItemGroup>
+
+  <!-- Publishing: By default, include PDB files in the package lib/* folder -->
+  <PropertyGroup Condition="'$(CreateSymbolPackage)' != 'true'">
+    <DebugSymbols>True</DebugSymbols>
+    <DebugType>portable</DebugType>
+    <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
+  </PropertyGroup>
```

Comment:
It doesn't look like ILRepack's `DebugInfo` option can handle newer (embedded) PDB formats; it can handle the Windows PE format (i.e. `full` or `pdbonly`) which the NuGet symbol server won't accept in a .snupkg for security reasons.

Below is what this looks like with the default (embedded) PDBs with ILRepack:

(`DebugType=embedded` + `DebugInfo=true`)
![embedded](https://github.com/videre-project/MTGOSDK/assets/32466081/af18855e-1ed7-4a16-bfc6-54581af0fcd0)

Instead, we have to bundle PDBs as part of the nupkg under the `lib/*` folder:

(`DebugType=portable` + `DebugInfo=false`)
![portable](https://github.com/videre-project/MTGOSDK/assets/32466081/2f2b9060-acce-4af9-9039-20e3eda3a952)